tests/Aspire.Hosting.Tests/DistributedApplicationTests.cs

```diff
@@ -386,6 +386,74 @@ public async Task VerifyDockerWithBoundVolumeMountWorksWithRelativePaths()
         await app.StopAsync();
     }
 
+    [LocalOnlyFact("docker")]
+    public async Task KubernetesHasResourceNameForContainersAndExes()
```

Comment:
It's too now, but you could write a unit test for this instead of integration. There is now an ApplicationExecutorTests file.

Reply:
Ooh, I'll need to remember that for future testing.